client/my-sites/theme/main.jsx

```diff
@@ -933,7 +933,14 @@ class ThemeSheet extends Component {
 						? this.appendSelectedStyleVariationToUrl( getUrl( this.props.themeId ) )
 						: null
 				}
-				onClick={ this.onButtonClick }
+				onClick={ () => {
+					this.props.recordTracksEvent( 'calypso_theme_sheet_primary_button_click', {
```

Comment:
We don't move this event to here: https://github.com/Automattic/wp-calypso/blob/e2d02bfb5c2e2510cc1c1a119dfd176a790dc62b/client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/events.ts, because this is not the pattern assembler event, am I right?

Reply:
That's right, this is unrelated to the pattern assembler 🙂